test/e2e/common/node/configmap.go

```diff
@@ -141,7 +141,7 @@ var _ = SIGDescribe("ConfigMap", func() {
 		gomega.Expect(err).To(gomega.HaveOccurred(), "created configMap %q with empty key in namespace %q", configMap.Name, f.Namespace.Name)
 	})
 
-	ginkgo.It("should update ConfigMap successfully", func(ctx context.Context) {
+	framework.ConformanceIt("should update ConfigMap successfully", f.WithNodeConformance(), func(ctx context.Context) {
```

Comment:
wouldn't the `ConformanceIt` require the special comment (with the release and such) and an update of conformance.yaml file?

Reply:
@SergeyKanzhelev the verify scripts should fail and indicate that - `require the special comment (with the release and such) and an update of conformance.yaml file?`